packages/ui-tooltip/src/Tooltip/types.ts

```diff
@@ -29,16 +29,21 @@ import type {
   PositionMountNode
 } from '@instructure/ui-position'
 
-export type ChildrenArgs = {
+export type TooltipRenderChildrenArgs = {
```

Comment:
Renamed this, because I think it's best if we prefix the globally exported component types.